.github/workflows/tests.yml

```diff
@@ -45,36 +45,84 @@ jobs:
             experimental: false
     container: ${{ matrix.container }}
     continue-on-error: ${{ matrix.experimental }}
+    outputs:
+      release: ${{ steps.push.outputs.release }}
     steps:
       - name: Check out head
         uses: actions/checkout@v2
         with:
           persist-credentials: false
+
       - name: Set up Python
         if: matrix.os == 'macos-latest'
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
+
       - name: Install Python dependencies
         run: |
           pip install --no-cache-dir -r ./requirements.txt
           pip install --no-cache-dir -r ./tests/requirements.txt
+
       - name: Install Covalent
         run: pip install cova --pre
-      - name: Install SSH Plugin
-        run: pip install -e .
-      - name: Start Covalent dispatcher server
+
+      - name: Build SSH plugin
+        run: python setup.py sdist
+
+      - name: Validate plugin
+        run: |
+          VERSION="$(cat ./VERSION)"
+          cd dist
+          tar xzf covalent-ssh-plugin-${VERSION}.tar.gz
+          diff covalent-ssh-plugin-${VERSION}/covalent_ssh_plugin/ssh.py ../covalent_ssh_plugin/ssh.py
```

Comment:
this diff line doesn't do anything.  you should be comparing the directory contents.

Reply:
updated the diff to compare directories